File: Programming Basics with JS/T02 - First Steps in Coding/lecture/07. Project Creation.js
function project(input){
    let timeForOne = 3;
    let firstname = (input[0]);
    let cProjects = (input[1]);
    let hoursForXProjects = timeForOne * cProjects;

    console.log("The architect " + firstname + " will need " + hoursForXProjects + " hours to complete " + cProjects + " project/s.");
}
project(["George", "4"])

File: Programming Basics with Go/T02 - First Steps in Coding/lecture/07. Project Creation.go
package main

import "fmt"

func greeting1() {
	
	var architectName string
	fmt.Scanln(&architectName)
	var countProjects int
	fmt.Scanln(&countProjects)

	fmt.Printf("The architect %s will need %d hours to complete %d project/s.", architectName, countProjects * 3, countProjects)


}